test_single_controller_ppo.py

```diff
@@ -430,16 +351,16 @@ class TrainActorGroup(SPMDActorGroup):
     def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
 
-    def build_models(self, pretrain_model_name: str):
+    def build_models(self, pretrain_model_name: str, config: Any):
```

Comment:
nit: We don't need to pass `pretrain_model_name` anymore, right? We can just get that from the config?